main/src/ca/uwaterloo/flix/language/phase/jvm/JvmOps.scala

```diff
@@ -730,6 +731,7 @@ object JvmOps {
     case MonoType.Int64 => Type.Int64
     case MonoType.BigInt => Type.BigInt
     case MonoType.Str => Type.Str
+    case MonoType.Region => Type.mkRegion(Type.Unit, SourceLocation.Unknown) // hack
```

Comment:
This is, as the comment suggests, a hack. But I think it's OK?

Reply:
This seems reasonable. This is really an ugly hack that should be removed by <someone>